salesforce/9.8/modules/ROOT/pages/salesforce-connector-studio.adoc

```diff
@@ -233,38 +233,49 @@ image::salesforce-mutual-tls.png[Mutual TLS]
+The Mule app controls the lifecycle connections. When the app determines that a given connection is not needed anymore, it checks the setting of *Disable session invalidation*. When the setting is `false` (default), the connector automatically destroys the connection for the session. To prevent a session from closing in this case, you can set the *Disable session invalidation* field to `true` or provide a function expression.
 
 Salesforce uses the same session for all your threads, so for example, if your session is active and you log in again, Salesforce uses the existing session instead of creating a new one.
 
-If the *Disable session invalidation* field is set to False, the connector automatically destroys the session after it's no longer needed.
+If the *Disable session invalidation* field is set to `false`, the connector automatically destroys the session after it's no longer needed.
 
-You should keep the session alive when you are working with threads or concurrency in general. Salesforce uses the same session for all your threads (for example, if you have an active session and you log in again, Salesforce uses the existing session instead of creating a new one). To make sure the connection doesn't close when a thread is finished, you should set the *Disable session invalidation* field to True in the Connection section of the connector's global element properties.
+You should keep the session alive when you are working with threads or concurrency in general. Salesforce uses the same session for all your threads (for example, if you have an active session and you log in again, Salesforce uses the existing session instead of creating a new one). To make sure the connection doesn't close when a thread is finished, you should set the *Disable session invalidation* field to `true` in the *Connection* section of the connector's global element properties.
 
 image::salesforce-disable-session.png[Disable Session Field]
 
 [[keepalive]]
-== Keep Alive feature (Deprecated - was removed in Salesforce 10.x)
+== Keep Alive Feature (Deprecated in Salesforce 10.x)
 
-Through this mechanism a scheduler can be set to make one simple API request (get server timestamp), with a specified time frame, in order to keep the session alive.
-To activate this feature, the Advanced reconnection params needs to be configured. This group contains two parameters:
+To keep a session alive, you can set a scheduler to make one simple API request (get server timestamp), within a specified timeframe.
+To activate this feature, you must configure the reconnection parameters in the *Advanced* tab, which include:
```

Comment:
"Advanced reconnection params" it is found in the General tab. 
![image](https://user-images.githubusercontent.com/42573217/87020519-9f6e6700-c1dc-11ea-9d0b-96f1b4967fcf.png)